core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -59,6 +62,7 @@ export class NavigationComponent implements OnInit {
   @Input() public autoSaveState: string = "";
   @Input() public currentWorkflowName: string = ""; // reset workflowName
   @Input() public particularVersionDate: string = ""; // placeholder for the metadata information of a particular workflow version
+  @Input() public particularExecutionName: string = "";
```

Comment:
make sure it is in a label fields, not a changeable input field, to disable any changes

Reply:
done